src/satellite/test.js

```diff
@@ -75,8 +75,8 @@ describe('Satellite()', () => {
     token = createToken({ sub: 'test-user@email.com' });
   });
 
-  afterEach(async () => {
-    await service.stop();
+  afterEach(() => {
```

Comment:
This is wrong, you need to return the Promise, so Jest waits:

```js
afterEach(() => service.stop())
```

Reply:
Right, I overlooked those brackets